src/Maestro/Client/src/MaestroApiOptions.cs

```diff
@@ -10,6 +11,8 @@ namespace Microsoft.DotNet.Maestro.Client
 {
     public partial class MaestroApiOptions
     {
+        public static readonly string AUTH_CACHE = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".maestro");
```

Comment:
Let's do `.darc` because people would mostly use darc locally with the interactive flow

Reply:
Also I think this folder already exists